internal/ovirt/resource_ovirt_vm.go

```diff
@@ -377,13 +386,28 @@ func handleVMPlacementPolicy(
 		}
 		hasPlacementPolicy = true
 	}
+
 	if hasPlacementPolicy {
+		isSupported, err := client.SupportsFeature(ovirtclient.FeaturePlacementPolicy)
+		if err != nil {
+			diags = append(diags, errorToDiag("check feature support", err))
+			return diags
+		}
+		if !isSupported {
+			diags = append(diags, diag.Diagnostic{
+				Severity: diag.Error,
```

Comment:
This should be a warning and the placement policy should not be applied.

Reply:
True, fixed it.